addons/planner_project/data/planner_data.xml

```diff
@@ -0,0 +1,1211 @@
+                                            <option value="customer_service">Customer Service</option>
+                                            <option value="repair_workshop">Repair Workshop</option>
+                                        </select>
+                                    </div>
+                                </li>
+                                <li class="row"><span>#1</span>
+                                    <input class="col-xs-12 col-sm-4 user_project" data-id="input_element_user_stage0" id="input_element_stage_0" type="text" value="Ideas"/>
+                                    <input class="col-xs-12 col-sm-7 col-sm-offset-1" id="input_element_stage_7" type="text" value="Idea is fully explained"/>
+                                </li>
+                                <li class="row">
+                                    <span>#2</span>
+                                    <input class="col-xs-12 col-sm-4 user_project" data-id="input_element_user_stage1" id="input_element_stage_1" type="text" value="To Do"/>
+                                    <input class="col-xs-12 col-sm-7 col-sm-offset-1" id="input_element_stage_8" type="text" value="Idea has been transformed into concrete actions"/>
+                                </li>
+                                <li class="row">
+                                    <span>#3</span>
+                                    <input class="col-xs-12 col-sm-4 user_project" data-id="input_element_user_stage2" id="input_element_stage_2" type="text" value="Done"/>
+                                    <input class="col-xs-12 col-sm-7 col-sm-offset-1" id="input_element_stage_9" type="text" value="Task is completed"/>
+                                </li>
+                                <li class="row">
+                                    <span>#4</span>
+                                    <input class="col-xs-12 col-sm-4 user_project" data-id="input_element_user_stage3" id="input_element_stage_3" type="text" value="Cancelled"/>
+                                    <input class="col-xs-12 col-sm-7 col-sm-offset-1" id="input_element_stage_10" type="text" value="Reason for cancellation has been documented"/>
+                                </li>
+                                <li class="row">
+                                    <span>#5</span>
+                                    <input class="col-xs-12 col-sm-4 user_project" data-id="input_element_user_stage4" id="input_element_stage_4" type="text"/>
+                                    <input class="col-xs-12 col-sm-7 col-sm-offset-1" id="input_element_stage_11" type="text"/>
+                                </li>
+                                <li class="row">
+                                    <span>#6</span>
+                                    <input class="col-xs-12 col-sm-4 user_project" data-id="input_element_user_stage5" id="input_element_stage_5" type="text"/>
```

Comment:
data-id is used to put current value to responsibility 'kanban stage'.Anyway i have make code working by removing data-id attr.